docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -273,6 +273,16 @@
         or dfuwu-archiver as that is a value limitation, you can choose to
         either have the service or not (set the 'disabled' value to
         true).</para>
+
+        <para>Any service related info needs to be under a service object, for
+        example: </para>
+
+        <para><programlisting>sasha:
+  wu-archiver:
+    service:
+       servicePort: 8877
+</programlisting>This also applies to the ESP services consistent with the
```

Comment:
I would say just "Roxie" not "the Roxie", there's not necessarily a single roxie in a setup.

But  more generally all components that have service definitions follow the same pattern.
I think confusing to say here, under sasha documentation. When it applies to esp, dali, dafilesrv and sasha. And, roxie's specification is slightly different, in that it has it's service ports defined under "roxieservice", of which there can be multiple. (see schema).